### DISCUSSION TITLE
Capacitor - change server between Prod, Beta and Dev

### DISCUSSION THREAD

**Discussion by developerSave** [quasarframework/quasar#14956] [CLI - Capacitor / Cordova modes]
Hi Everyone,
I'm trying to develop my first Android App using Quasar-Capacitor-VueJS2 and I have a problem:
I need to switch my app server among 3 environments and I don't know if I've to change   "  devServer: {} " in my quasar.config.js file or have I to change server in other files?

Thank you,
Lorenzo

**Comment by MilosPaunovic** [comment]
There is no need to duplicate - you should focus on #14954.
